src/test/resources/com/cloudbees/jenkins/support/filter/ContentMappingsTest/operatingSystemIncludedAsStopWord/com.cloudbees.jenkins.support.filter.ContentMappings.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<com.cloudbees.jenkins.support.filter.ContentMappings resolves-to="com.cloudbees.jenkins.support.filter.ContentMappings$XmlProxy">
+  <stopWords>
+    <string>authenticated</string>
+    <string>admin</string>
+    <string>mac</string>
+    <string>unknown</string>
+    <string>computer</string>
+    <string>view</string>
+    <string>linux</string>
```

Comment:
The new ones shouldn't be here so that the test proves that even when the stop words are loaded from disk, the new stop words are taken into account.

Reply:
Maybe add a comment?